### PR TITLE
More lazy string generation for debug logging

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review.

### CODE REVIEW

File: docproc/src/main/java/com/yahoo/docproc/jdisc/DocumentProcessingHandler.java [hunk 1]
```diff
@@ -200,7 +200,7 @@ private LaterTimerTask(DocumentProcessingTask processingTask, long delay) {
 
         @Override
         public void run() {
-            log.log(Level.FINE, "Submitting after having waited " + delay + " ms in LATER queue: " + processingTask);
+            log.log(Level.FINE, () -> "Submitting after having waited " + delay + " ms in LATER queue: " + processingTask);
```
Comment (jonmv) [inline docproc/src/main/java/com/yahoo/docproc/jdisc/DocumentProcessingHandler.java:203]: `(((＼（✘෴✘）／)))`

**Comment by jonmv** [comment]
Or @bratseth :)

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java [hunk 2]
```diff
@@ -579,8 +579,7 @@ public StatusPageResponse fetchStatusPage(StatusPageServer.HttpRequest httpReque
             responseCode = StatusPageResponse.ResponseCode.INTERNAL_SERVER_ERROR;
             message = "Internal Server Error";
             hiddenMessage = ExceptionUtils.getStackTraceAsString(e);
-            log.log(Level.FINE, "Unknown exception thrown for request " + httpRequest.getRequest() +
-                    ": " + hiddenMessage);
+            log.log(Level.FINE, "Unknown exception thrown for request " + httpRequest.getRequest() + ": " + hiddenMessage);
```
Comment (bjorncs) [inline clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java:582]: Missing lazy logging and generating of stack trace

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java [hunk 3]
```diff
@@ -52,24 +52,19 @@ public void handleAllDistributorsInSync(final ClusterState currentState,
                                             final DatabaseHandler database,
                                             final DatabaseHandler.Context dbContext) throws InterruptedException {
         int startTimestampsReset = 0;
-        log.log(Level.FINE, String.format("handleAllDistributorsInSync invoked for state version %d", currentState.getVersion()));
+        log.log(Level.FINE, "handleAllDistributorsInSync invoked for state version %d", currentState.getVersion());
```
Comment (bjorncs) [inline clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java:55]: Missing lazy

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java [hunk 4]
```diff
@@ -52,24 +52,19 @@ public void handleAllDistributorsInSync(final ClusterState currentState,
...
-                                    node, nodeState.getStartTimestamp()));
-                        }
+                        log.log(Level.FINE, () -> String.format("Storing away new start timestamp for node %s (%d)", node, nodeState.getStartTimestamp()));
                         nodeInfo.setStartTimestamp(nodeState.getStartTimestamp());
                     }
                     if (nodeState.getStartTimestamp() > 0) {
-                        if (log.isLoggable(Level.FINE)) {
-                            log.log(Level.FINE, String.format("Resetting timestamp in cluster state for node %s", node));
-                        }
+                        log.log(Level.FINE, "Resetting timestamp in cluster state for node %s", node);
```
Comment (bjorncs) [inline clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java:67]: Was lazy but no longer?

**Review by bjorncs — APPROVED** [review]

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java (hunk 2 above)
Comment (jonmv) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java:582]: Yes, I gave up on a very few of these :)

Comment (jonmv) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/FleetController.java:582]: Could have used the if thing. sure. Ok ok .

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java (hunk 3 above)
Comment (jonmv) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java:55]: This is actually lazy.

File: clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java (hunk 4 above)
Comment (jonmv) [reply clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/StateChangeHandler.java:67]: It actually still is.

**Review by bjorncs — APPROVED** [review]
:shipit:

**Pull request merged**